### PR TITLE
Included CSS changes for view grade, reverted sidebar logo, slight change to course page

### PR DESCRIPTION
**Pull request by SanjithSenthil**
- This pull request is for css changes to  view grade, sidebar, edit user details and course page.
- Included sidebar in the above pages

### CODE REVIEW

File: flask/website/templates/EditDetails.html [hunk 1]
```diff
@@ -51,65 +61,67 @@
...
 
-         <!-- Date of Birth (DOB) Field -->
-         <label for="dob">Date of Birth (DOB):</label>
-         <input type="date" id="dob" name="dob" >
-         <br><br>
+            <!-- First Name Field -->
+            <label for="firstName">First Name:</label>
+            <input type="text" id="firstName" name="firstName" >
 
-        <!-- Password Field -->
```
Comment (PaintedW0lf) [inline flask/website/templates/EditDetails.html:76]: is password field getting deleted?

**Review by PaintedW0lf — CHANGES_REQUESTED** [review]
Please look at the comments and if you could just recheck them as i am not sure if certain things should have changed

**Comment by SanjithSenthil** [comment]
Replied to the comments

**Review by PaintedW0lf — APPROVED** [review]
Reviewed! Works with consistency

**Pull request merged**